tests/containers/bci_version_check.pm

```diff
@@ -25,6 +25,7 @@ sub run {
 
     my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
     my $build = get_required_var('CONTAINER_IMAGE_BUILD');
+    my ($buildrelease) = $build =~ /-(.+)$/;
```

Comment:
The request was to grep from the latest dash, as we could introduce another one (after the SLES version).
Also here `$buildrelease` will be just a boolean indicating if the regex matched or not.

Reply:
```perl
my @build = split(/-/, $build);
my $buildrelease = @build[-1];
```
Maybe?

Reply:
I implemented this suggestion. Let's see the VRs.